test/soak/serve_hostnames/README.md

```diff
@@ -0,0 +1,168 @@
+# Soak Test serve_hostnames
+This directory contains the source for a soak test `server_hostnames` which performs the following actions when used with the GCE provider:
```

Comment:
Fixed, thanks. Every single serve_hostnames was originally typed as server_hostnames and then fixed up. I missed this one.